### ISSUE TITLE
Another nice to have

### ISSUE THREAD

**Issue by hkirsman** [wunderkraut/WunderTools#23]
Could we prompt user for some basic info on first vagrant up?
What's your virtual machine name (default ansibleref):
What ip would you want set on your vm (default 192.168.10.170): 
Do you want varnish support (default yes): 
etc

**Comment by tharna** [comment]
Yes, there is plans for creating a wizard tool for creating a new project in the first place and also some user specific preferences that can be used to customize the per user DX

**Comment by badrange** [comment]
Would it make sense to use something like Yeoman for this? Not exactly sure how yeoman works, but it would be cool to type yo ```product-name``` and then go from there

**Comment by hkirsman** [comment]
That's even better!

**Comment by fredo-wunder** [comment]
@badrange i gave ½ an hour to yeoman to test it out, but haven't succeded :(. maybe one should just try more, but maybe we can have a chat on how to do that.

**Comment by fredo-wunder** [comment]
so if anyone feels like testing this, it should work by: 

1: cloning git@github.com:fredo-wunder/generator-wunderproject.git
2: installing that thing
3: launching yo generator-wunderproject 
4: profit, hopefully... if not, bug me ! :)

**Comment by hkirsman** [comment]
2: installing that thing ? :)

**Comment by fredo-wunder** [comment]
npm install from generator-wunderproject

:)

**Comment by fredo-wunder** [comment]
i remembered wrong... 

to launch it, use `yo wunderproject`

just tested it, looks like quite ok!
